packages/flutter/lib/src/material/app.dart

```diff
@@ -132,9 +132,23 @@ class MaterialApp extends StatefulWidget {
        ),
        super(key: key);
 
-  /// A one-line description of this app for use in the window manager.
+  /// A one-line description used by the device to identify the app for the user.
```

Comment:
maybe mention the window manager in a second paragraph? or talk about specifics of where this is used on Android and iOS?

Reply:
I didn't think "window manager" was a term that offered much to mobile devs, but explaining what exactly iOS and Android do with title would help.